doc/doxygen/Makefile

```diff
@@ -1,7 +1,7 @@
 RIOTBASE=$(shell git rev-parse --show-toplevel)
 # Generate list of quoted absolute include paths. Evaluated in riot.doxyfile.
 export STRIP_FROM_INC_PATH_LIST=$(shell \
-    git ls-tree -dr --full-tree --name-only HEAD |\
+    git ls-tree -dr --full-tree --name-only HEAD core drivers sys |\
```

Comment:
Just as a reminder. `STRIP_FROM_INC_PATH_LIST` doesn't control if something is in the documentation or not. It just influences how '#include path/to/somewhere.h' entries **inside** the doc are generated. [Here](https://github.com/RIOT-OS/RIOT/issues/4794#issuecomment-188270523) is the story behind it ( with a pathetic mistake, writing '#define' instead of  '#include':D)